recipes/git-python/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "git-python" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/pynickle/git-python
+
+build:
+  skip: true  # [win]
```

Comment:
This one can be noarch.

It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
made noarch instead of skipping windows